101-asp-net-app-service/metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+    "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
+    "type": "QuickStart",
+    "itemDisplayName": "ASP.NET app",
+    "description": "This template allows you to create an App Service for deploying a ASP.NET app.", 
+    "summary": "Deploy an App Service that supports ASP.NET v4.0. It gives you the support to deploy your own ASP.NET app.",
+    "githubUsername": "jose-mart",
+    "tags":{
+      "subscriptiontype": "Azure4Student",
+      "resource":"app services",
+      "os": "windows",
+      "level": "Begginer",
+      "cost": "$0"

```

Comment:
zero?

Reply:
Yes ... It was supposed that "$0" was for a free tier service.